rr_arduino/imu.ino

```diff
@@ -1,7 +1,7 @@
 
 //ROS publisher commands
-ros::Publisher imu_pub(&imu_msg);
-ros::Publisher magnetic_pub(&magnetic_msg);
+ros::Publisher imu_pub("/arduino/vehicle_state"&imu_msg);
+ros::Publisher magnetic_pub("/arduino/enc_vel"&magnetic_msg);
```

Comment:
?

Reply:
Before the change, the arduino kept giving an error saying "no call for function" in both of those lines. I added those and it worked. I am not sure of its significance and why it worked so I left it there for now.

Reply:
add a comment between the publisher name and the _msg function

Reply:
I meant a comma, sorry about that. like in this http://wiki.ros.org/rosserial_arduino/Tutorials/Hello%20World